.ci/preview-environments/charts/c8sm/templates/ingress.yml

```diff
@@ -31,7 +31,7 @@ spec:
         pathType: Prefix
       - backend:
           service:
-            name: {{ template "core.fullname" $camundaPlatform }}
+            name: {{ template "core.fullname" $camundaPlatform }}-gateway
```

Comment:
New (separate) service is introduced in https://github.com/camunda/camunda-platform-helm/pull/3791.